cmd/webhook/main.go

```diff
@@ -75,14 +69,6 @@ func main() {
 		logger.Fatal("Failed to get the client set", zap.Error(err))
 	}
 
-	buildClient, err := buildclientset.NewForConfig(cfg)
-	if err != nil {
-		log.Fatalf("Error building Build clientset: %s", err.Error())
-	}
-
-	kubeInformerFactory := kubeinformers.NewSharedInformerFactory(kubeClient, time.Second*30)
-	bldr := onclusterbuilder.NewBuilder(kubeClient, kubeInformerFactory, logger)
-
 	pkgoptions := pkgwebhook.ControllerOptions{
```

Comment:
Same here, these can just be `options` and `controller` now since there's nothing to distinguish from.